appeals/web/src/server/appeals/appeal-details/representations/lpa-statement/incomplete/incomplete.mapper.js

```diff
@@ -11,11 +13,39 @@ export function rejectLpaStatementPage(appealDetails) {
 
 	const pageContent = {
 		heading: 'Why is the statement incomplete?',
-		backLinkUrl: `/appeals-service/appeal-details/${appealDetails.appealId}/lpa-statement/incomplete/confirm`,
+		backLinkUrl: `/appeals-service/appeal-details/${appealDetails.appealId}/lpa-statement`,
 		preHeading: `Appeal ${shortReference}`,
 		hint: 'Select all that apply.',
 		headingClasses: 'govuk-heading-l'
 	};
 
 	return pageContent;
 }
+
+/**
+ * @param {import('got').Got} apiClient
+ * @param {Appeal} appealDetails
+ * @returns {Promise<PageContent>}
+ * */
+export async function setNewDatePage(apiClient, appealDetails) {
+	const extendedDeadline = await addBusinessDays(apiClient, new Date(), 7);
```

Comment:
I'd be tempted to pull this up to the controller layer rather than having the asynchronicity trickle down but no biggie

Reply:
Good shout, updated accordingly!